### PR TITLE
Add blogpost on Mitiq

### PR DESCRIPTION
**Pull request by nathanshammah**
@willzeng feel free to directly push commits for edits.

### CODE REVIEW

File: posts/mitiq.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+
+
+<p dir="ltr"><p>We&rsquo;re excited to introduce <a href="https://www.github.com/unitaryfund/mitiq"><strong>Mitiq</strong></a>, a toolchain that helps reduce the errors occuring in noisy quantum computers. </p>
+<br>
+
+<p>Mitiq's white paper has just been uploaded on the <a href="https://arxiv.org/abs/2009.04417"><strong>arXiv</strong></a>. In it, we introduce the library and guide first users through its main features. Mitiq is designed to be a thin layer between the quantum programmers (and their circuits) and their real quantum processors of choice, or simulators.</p>
+<br>
+
+<p>In the paper we show how to improve the performance of real quantum processors, running quantum programs on publicly available quantum computers on the cloud, Rigetti Computing and IBM Q processors.  </p>
```
Comment (karalekas) [inline posts/mitiq.html:92]:
```suggestion
<p>In the paper we show how to improve the performance of real quantum processors, running quantum programs on publicly available quantum computers on the cloud (e.g. Rigetti Computing and IBM Q processors).  </p>
```

File: posts/mitiq.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+<br>
+
+<p>To begin with, we've focused on zero-noise extrapolation, an error mitigation technique that has already been tested in the lab. We illustrate how zero-noise extrapolation can be applied even if a user has no direct access to pulse-level control over the compilation of a quantum circuit on a processor, but instead can implement <a href="https://arxiv.org/abs/2005.10921"><strong>digital zero noise extrapolation</strong></a>.</p>
```
Comment (karalekas) [inline posts/mitiq.html:95]:
```suggestion
<p>To begin with, we've focused on zero-noise extrapolation, an error mitigation technique that has already been tested in the lab. We illustrate how zero-noise extrapolation can be applied even if a user has no direct access to pulse-level control over the compilation of a quantum circuit on a processor. We call this method <a href="https://arxiv.org/abs/2005.10921"><strong>digital zero-noise extrapolation</strong></a>.</p>
```

File: posts/mitiq.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+<br>
+
+<p>If you're interested in Mitiq and in quantum error mitigation, you can find information on usage and additional references in <strong>Mitiq's documentation</strong> at <a href="https://mitiq.readthedocs.io/en/stable/"><strong>mitiq.readthedocs.io</strong></a>.  </p>
```
Comment (karalekas) [inline posts/mitiq.html:98]:
```suggestion
<p>If you're interested in Mitiq and in quantum error mitigation, you can find information on usage and additional references in <a href="https://mitiq.readthedocs.io/en/stable/"><strong>Mitiq's documentation</strong></a>.  </p>
```

File: posts/mitiq.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                   class="button heavy square-button">
+        <!--</div>-->
+        </div>
+    </form>
+    </div>
+    </center>
+
+    <!--End mc_embed_signup-->
+<br>
+<p>We're looking forward to the feedback of the quantum software community: from fixing bugs to proposing enhacements in zero-noise extrapolation to additional quantum error-mitigation techniques. We're keeping a list of <a href="https://github.com/unitaryfund/mitiq/wiki/Mitiq-Talks-and-Events"><strong>events</strong></a> where members of the Unitary Labs team will be attending.</p>
```
Comment (karalekas) [inline posts/mitiq.html:128]:
```suggestion
<p>We're looking forward to the feedback of the quantum software community: from fixing bugs to proposing enhacements in zero-noise extrapolation to additional quantum error mitigation techniques. We're also keeping a list of <a href="https://github.com/unitaryfund/mitiq/wiki/Mitiq-Talks-and-Events"><strong>events</strong></a> where members of the Unitary Labs team will be attending.</p>
```

File: posts/mitiq.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+<p>In the paper we show how to improve the performance of real quantum processors, running quantum programs on publicly available quantum computers on the cloud, Rigetti Computing and IBM Q processors.  </p>
+<br>
+
+<p>To begin with, we've focused on zero-noise extrapolation, an error mitigation technique that has already been tested in the lab. We illustrate how zero-noise extrapolation can be applied even if a user has no direct access to pulse-level control over the compilation of a quantum circuit on a processor, but instead can implement <a href="https://arxiv.org/abs/2005.10921"><strong>digital zero noise extrapolation</strong></a>.</p>
+<br>
+
+<p>If you're interested in Mitiq and in quantum error mitigation, you can find information on usage and additional references in <strong>Mitiq's documentation</strong> at <a href="https://mitiq.readthedocs.io/en/stable/"><strong>mitiq.readthedocs.io</strong></a>.  </p>
+<br>
+
+If you'd like to keep up to date with Mitiq's development, you can sign up to Unitary Fund's mailing list
```
Comment (karalekas) [inline posts/mitiq.html:101]:
```suggestion
<p>If you'd like to keep up to date with Mitiq's development, you can sign up to Unitary Fund's mailing list below!</p><br>```

File: posts/mitiq.html [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    </form>
+    </div>
+    </center>
+
+    <!--End mc_embed_signup-->
+<br>
+<p>We're looking forward to the feedback of the quantum software community: from fixing bugs to proposing enhacements in zero-noise extrapolation to additional quantum error-mitigation techniques. We're keeping a list of <a href="https://github.com/unitaryfund/mitiq/wiki/Mitiq-Talks-and-Events"><strong>events</strong></a> where members of the Unitary Labs team will be attending.</p>
+<br>
+<p>Best wishes,</p>
+<p>Unitary Labs – the Engineering team at Unitary Fund (Andrea Mari, Nathan Shammah, Peter Karalekas, Ryan LaRose and Will Zeng)</p>
```
Comment (karalekas) [inline posts/mitiq.html:131]:
```suggestion
<p>Unitary Labs – the Engineering team at Unitary Fund
<br>(Andrea Mari, Nathan Shammah, Peter Karalekas, Ryan LaRose and Will Zeng)</p>
```

**Review by karalekas — CHANGES_REQUESTED** [review]
Just a couple small things -- looks great!

**Comment by willzeng** [comment]
Thanks! I made some updates and pushed to a new merged branch #31 and so will close this one. If there's any further comments on the post then please do open up a PR :)

**Pull request closed without merging**